e2e/fuzzing_test.go

```diff
@@ -49,7 +49,7 @@ var _ = Describe("E2E TEST:"+checks.CheckFuzzing, func() {
 			expected := scut.TestReturn{
 				Error:         nil,
 				Score:         checker.MaxResultScore,
-				NumberOfWarn:  0,
+				NumberOfWarn:  4,
 				NumberOfInfo:  0,
```

Comment:
This should probably be 3 warn and 1 info.

The test is broken right now though #2670